dev-docs/bidders/oraki.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: bidder
+title: Oraki
+description: Prebid Oraki Bidder Adapter
+biddercode: oraki
+gpp_sids: usstate_all
+tcfeu_supported: false
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+deals_supported: false
+floors_supported: true
+fpd_supported: false
+ortb_blocking_supported: false
+media_types: banner, video, native
+multiformat_supported: will-bid-on-one
+userIds: all
+pbjs: true
+pbs: true
```

Comment:
```suggestion
pbs: false
```

Then we can merge this and you can open another docs PR that changes this to `true`.